pkg/services/provisioning/values/values.go

```diff
@@ -19,11 +19,14 @@ import (
 	"github.com/grafana/grafana/pkg/util/errutil"
 )
 
+// IntValue represents a string value in a yaml
```

Comment:
I think it's best to stick with the acronym YAML, lest we get PRs to change it
```suggestion
// IntValue represents a string value in a YAML
```